README.md

```diff
@@ -94,42 +96,31 @@ If you want to have your generic foreign key represented as hyperlink, simply us
         tagged_object = serializers.GenericRelatedField({
             Bookmark: serializers.HyperlinkedRelatedField(view_name='bookmark-detail'),
             Note: serializers.HyperlinkedRelatedField(view_name='note-detail'),
-        }, read_only=True)
+        })
 
         class Meta:
-            model = Tag
+            model = TaggedItem
             exclude = ('id', )
+```
 
-The JSON representation of the same `Tag` example object could now look something like this:
+The JSON representation of the same `TaggedItem` example object could now look something like this:
 
+```json
     {
-        'tagged_object': '/bookmark/1/',
-        'tag_name': 'django'
+        "tagged_object": "/bookmark/1/",
+        "tag_name": "django"
     }
+```
 
-These examples cover the default behavior of generic foreign key representation. However, you may also want to write to generic foreign key fields through your API.
+## Writing to generic foreign keys
 
-By default, a `GenericRelatedField` iterates over its nested serializers and returns the value of the first serializer, that is actually able to perform `from_native` on the input value without any errors.
+The above `TagSerializer` is also writable. By default, a `GenericRelatedField` iterates over its nested serializers and returns the value of the first serializer that is actually able to perform `to_internal_value()` without any errors.
 Note, that (at the moment) only `HyperlinkedRelatedField` is able to serialize model objects out of the box.
```

Comment:
I'm not sure, but intuitively, the serializers you choose need to know the type of the model during deserialization, right?

HyperlinkRelatedField knows the type of the model (because it's implied from the URL). Other ones (e.g. `SlugRelatedField`, `PrimaryKeyRelatedField`) don't know the type of the model, they just know a slug or PK.

So I'd say this is likely to still be true.